includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -521,7 +519,7 @@ private function wc_setup_redirect_to_wc_admin_onboarding() {
 		$onboarding_data = get_option( 'wc_onboarding_profile', array() );
 		update_option( 'wc_onboarding_profile', array_merge( $onboarding_data, $profile_updates ) );
```

Comment:
Sorry for not realize it before, but using `wc_` prefix for options on WooCommerce isn't a good think, we only use `woocommerce_` prefix for options on core.
Also since it uses `wc_` prefix it doesn't get removed while running the [`uninstall.php`](https://github.com/woocommerce/woocommerce/blob/master/uninstall.php#L54).

@timmyc should be nice to fix those before merging WooCommerce Admin into core.

Reply:
Thanks for the review, @claudiosanches!  Addressed the changes mentioned above.

>should be nice to fix those before merging WooCommerce Admin into core.

I think they're all fixed in wc-admin and just needed to be updated here.  May be worthwhile to move some of this onboarding "reset" functionality into wc-admin so core doesn't need to know much about the logic behind each option; will follow-up on that in the upcoming weeks.